platform/pwl.c

```diff
@@ -276,15 +284,15 @@ pointer_on_enter (void* data,
                   wl_fixed_t fixed_x,
                   wl_fixed_t fixed_y)
 {
-    PwlDisplay *display = data;
-    if (pointer != display->pointer) {
+    PwlSeat *pwl_seat = data;
```

Comment:
There is no need for the `pwl_` prefix in this variable, you can just call it `seat`:

```c
PwlSeat *seat = data;
```

Reply:
Ah, wait, I see why you have done this: the `struct wl_seat` in some places is called `seat`… Because the code deals with a `PwlSeat` more often, I would call that `seat` and the `struct wl_seat` I would call it `wl_seat` when needed.

Reply:
I just pushed another version where I replace struct wl_seat *seat by  ` struct wl_seat *w_seat`. Probably your suggestion it's much better than mine.

Reply:
Finally, the PR updated with `s/w_seat/wl_seat/g`: https://github.com/Igalia/cog/commit/cf3f303efeaadee9358f158175ae23e33ae20d50